### PR TITLE
docs: vs code plugin pages PTL-1665

### PR DESCRIPTION
**Pull request by ivangenesis**
ATTENTION! NEW DOCS STRUCTURE
FOR THE NEXT THREE WEEKS, ONLY BRANCH FROM new-docs-structure
OTHERWISE, WAIT UNTIL THE NEW STRUCTURE HAS BEEN RELEASED

  - Please check the [internal contributions guide](https://www.notion.so/genesisglobal/Contributing-new-documentation-75953fb245f246ff872789035451a0c4) for details
  
__________

Thank you for contributing to the documentation.

Do the changes you have made apply to both Current and Previous versions?
Yes 

Have you done a trial build to check all new or changed links?
Yes 

Is there anything else you would like us to know?
No

**This week's exciting advice from the style guide**

- Write your headings in sentence case:

  - A good example
    This is a correct heading.
  - A Bad Example
    This is not a correct heading.

### CODE REVIEW

File: docs/001_develop/01_development-environment/005_vscode-plugin/index.mdx [hunk 1]
```diff
@@ -1,13 +1,168 @@
...
+1. To install the Genesis UI Generator, visit [Genesis UI Generator on Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=GenesisGlobal.genesis-ui-generator). Click on **Install** or search for `Genesis UI Generator` in the extensions panel in VS Code.
+
+2. After installing the extension, the Genesis icon will be visible in the VS Code side menu on the left.
+
+![Genesis UI Generator Icon](/img/vscode-extension/icon-extension.png)
+
+## Configuration
+
+- Open your Genesis project (root folder) in VS Code before starting to use the extension features.
+- Ensure that the source folder path (`src`) is correctly configured in the VS Code extension's settings.
```
Comment (matteematt) [inline docs/001_develop/01_development-environment/005_vscode-plugin/index.mdx:38]: I know this is quite obvious for us but it's maybe worth explaining how to do that in detail as our target audiance are people who probably don't feel very comfortable configuring tools

**Review by matteematt — APPROVED** [review]
Overall looks great. Just a small point I think it might be worth making clear that this doesn't support angular and react yet

**Review by wjhendry — APPROVED** [review]

File: docs/001_develop/01_development-environment/005_vscode-plugin/index.mdx (hunk 1 above)
Comment (ivangialorenco) [reply docs/001_develop/01_development-environment/005_vscode-plugin/index.mdx:38]: makes sense, will add more details on it.

Comment (ivangenesis) [reply docs/001_develop/01_development-environment/005_vscode-plugin/index.mdx]: Done. updated

**Comment by ivangenesis** [comment]
Updated with more details and no angular/react support. please @matteematt and @wjhendry check it out.

File: docs/001_develop/01_development-environment/005_vscode-plugin/index.mdx [hunk 2]
```diff
@@ -1,13 +1,173 @@
...
+2. Click on `Advanced Config` to configure your generation:
+
+![Genesis View Filled](/img/vscode-extension/genesis-view-filled.png)
+
+- `Route` creates a new route component with all the files (ts, template, styles) for the new custom UI.
+- `Custom Navigation Bar/Header` generates HTML components for a header or navigation bar.
+- `Custom Grid Styles` generates a file that provides custom colours and style for grids in your Genesis application.
+- `Design Tokens` generates or replaces the **design-tokens.json** file, managing the colours and theme of your entire application.
+
+PS: Genesis View doesn't support React or Angular generation yet.
```
Comment (matteematt) [inline docs/001_develop/01_development-environment/005_vscode-plugin/index.mdx:91]:
```suggestion
:::tip
Genesis View doesn't support React or Angular generation yet.
:::
```

**Review by matteematt — APPROVED** [review]

**Pull request merged**